project/Dependencies.scala

```diff
@@ -131,7 +136,7 @@ object Dependencies {
     workbenchGoogle2Test,
     workbenchOpenTelemetry,
     workbenchOpenTelemetryTest,
-    "net.logstash.logback" % "logstash-logback-encoder" % "6.2", // for structured logging in logback
+//    "net.logstash.logback" % "logstash-logback-encoder" % "6.2", // for structured logging in logback
```

Comment:
This isn't needed for logstash? (I don't know where it came from originally)

Reply:
think I added it a while ago for logging things in json format..since we're not doing that, we don't need this in...I didn't delete it just for future reference